setup.py

```diff
@@ -60,8 +58,11 @@ def find_data_files():
         'console_scripts': [
             'mopt = metaopt.core.cli:main',
             ],
+        'OptimizationAlgorithm': [
+            'random = metaopt.algo.random:Random',
+            ],
         },
-    install_requires=['six', 'PyYAML', 'pymongo>=3'],
+    install_requires=['PyYAML', 'pymongo>=3', 'numpy', 'scipy'],
```

Comment:
I assume the previous build tox stopped here

Reply:
This was not the case. I will reproduce it live to demo the case to you.